telemetry/prometheus/prometheus.yml

```diff
@@ -7,4 +7,4 @@ scrape_configs:
     metrics_path: /metrics
     static_configs:
       - targets:
-          - 'openfga:2112'
\ No newline at end of file
+          - 'otel-collector:1234'
```

Comment:
Can we keep this more consistent with the default metrics port? Maybe `2113` instead of `2112`?

Reply:
Sounds good